packages/tasks/src/model-libraries.ts

```diff
@@ -465,12 +465,6 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Tencent/HunyuanDiT",
 		countDownloads: `path:"pytorch_model_ema.pt" OR path:"pytorch_model_distill.pt"`,
 	},
-	"hunyuan3d-2": {
-		prettyLabel: "Hunyuan3D-2",
-		repoName: "Hunyuan3D-2",
-		repoUrl: "https://github.com/Tencent/Hunyuan3D-2",
-		countDownloads: `path_filename:"model_index" OR path_filename:"config"`,
-	},
```

Comment:
Was this intentional? I think we just wanted to remove the new snippet, not the library. Downloads are no longer tracked [here](https://huggingface.co/tencent/Hunyuan3D-2.1), and probably incomplete [here](https://huggingface.co/tencent/Hunyuan3D-2) (there's a config.json at the root level in the second model).